VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleListEntryController.cs

```diff
@@ -185,7 +177,7 @@ public IHttpActionResult DeleteLinks(webModel.ListEntryLink[] links)
             //Scope bound security check
             CheckCurrentUserHasPermissionForObjects(CatalogPredefinedPermissions.Delete, links);
 
-            InnerUpdateLinks(links, (x, y) => x.Links = x.Links.Where(l => string.Join(":", l.CatalogId, l.CategoryId) != string.Join(":", y.CatalogId, y.CategoryId)).ToList());
+            InnerUpdateLinks(links, (x, y) => x.Links = x.Links.Where(l => String.Join(":", l.CatalogId, l.CategoryId) != String.Join(":", y.CatalogId, y.CategoryId)).ToList());
             return StatusCode(HttpStatusCode.NoContent);
         }
 
```

Comment:
Do we need it? Our general rule is using string, int, etc. vs String, Int32, etc.

Reply:
done